code/_onclick/hud/action_button.dm

```diff
@@ -1,4 +1,5 @@
 #define ACTION_BUTTON_DEFAULT_BACKGROUND "default"
+#define ACTION_BUTTON_PAI_BACKGROUND "bg_tech"
```

Comment:
you dont need this, you use this define in only one place and that isnt even what actual pai backgrounds look like